### ISSUE TITLE
AP an loss are always 0

### ISSUE THREAD

**Issue by zsy7532** [stefanopini/simple-HRNet#110]
![image](https://github.com/stefanopini/simple-HRNet/assets/137871540/36a4745c-1b17-4add-a529-a5f8ee790faf)
I modified the number of keypoints to 3, and the parameters i set are :
self: <training.COCO.COCOTrain object at 0x7f21157035e0>
exp_name: 20240619_1104
ds_train: <datasets.COCO.COCODataset object at 0x7f2115703250>
ds_val: <datasets.COCO.COCODataset object at 0x7f21156fcf70>
epochs: 200
batch_size: 4
num_workers: 4
loss: JointsMSELoss
lr: 1e-05
lr_decay: True
lr_decay_steps: (170, 200)
lr_decay_gamma: 0.1
optimizer: Adam
weight_decay: 0.0001
momentum: 0.9
nesterov: False
pretrained_weight_path: /root/autodl-tmp/simple-HRNet/hrnet_w32-36af842e.pth
checkpoint_path: None
log_path: ./logs
use_tensorboard: True
model_c: 32
model_nof_joints: 3
model_bn_momentum: 0.1
flip_test_images: False
device: cuda:0
__class__: <class 'training.Train.Train'>
Please tell how to solve it?

**Comment by Weeknd0214** [comment]
标签问题
